src/integTest/resources/server-config/testCreateLibertyConfigDir.gradle

```diff
@@ -12,13 +12,11 @@ buildscript {
     }
     dependencies {
         classpath 'net.wasdev.wlp.gradle.plugins:liberty-gradle-plugin:1.1-SNAPSHOT'
-        classpath files('gradle/wlp-anttasks.jar')
     }
 }
 
 apply plugin: 'java'
 apply plugin: 'maven'
-apply plugin: 'war'
 apply plugin: 'liberty'
 
 sourceCompatibility = 1.8
```

Comment:
Setting sourceCompatibility and targetCompatibility to 1.8 should fail the travis ci test for oraclejdk7 configuration.  But for some reason your travis build is still passed.

You should change it back to 1.7 for all your test gradle file.